sdk/storage/azure-storage-blob-cryptography/pom.xml

```diff
@@ -101,7 +101,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-keys</artifactId>
-      <version>4.2.0-beta.6</version> <!-- {x-version-update;unreleased_com.azure:azure-security-keyvault-keys;dependency} -->
```

Comment:
@gapra-msft Storage requires `unreleased_KV_keys` dependency?

Reply:
This is a test dependency. As long as we have a version of keyvault that has an implementation of LocalKeyEncryptionKey we are good.

Reply:
This minor release does not include local cryptography, however, master will not be affected by this. Storage can keep using the latest Key Vault beta version. I will update said beta version after this release is done.